internal/flow/handler_test.go

```
@@ -579,3 +580,152 @@ func TestDeleteFlow(t *testing.T) {
+		{
+			name: "Success",
+			requestBody: createFlowStepRequest{
+				Protocol: "http",
+				Target:   "target",
+				Current:  validUUID,
+				Next:     validUUID,
+			},
+			pathID: "1",
+			mockSetup: func(m *mockFlowQuerier) {
+				ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					json.NewEncoder(w).Encode([]serviceDependency{
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)